ethcore/engines/authority-round/src/lib.rs

```diff
@@ -970,11 +975,15 @@ impl AuthorityRound {
 	/// Drops all `EmptySteps` less than or equal to the passed `step`, irregardless of the parent hash.
 	fn clear_empty_steps(&self, step: u64) {
 		let mut empty_steps = self.empty_steps.lock();
-		*empty_steps = empty_steps.split_off(&EmptyStep {
+		if empty_steps.is_empty() {
+			return
```

Comment:
```suggestion
			return;
```

Super nit, just personal preference but I prefer statements (rather than expressions) to always end with `;`.